static/js/src/advantage/users/api.ts

```diff
@@ -55,25 +49,17 @@ const accountUserRequestInit: RequestInit = {
 const getAccountUserRequestUrl = (accountId: string, urlParams: string) =>
   `/advantage/accounts/${accountId}/user${urlParams}`;
 
-type JSONResponse = {
-  data?: Record<string, unknown>;
-  error?: string;
-  errors?: string;
-};
-
-const handleResponse = async (response: Response): Promise<JSONResponse> => {
-  const responseJson: JSONResponse = await response.json();
+const handleResponse = async (response: Response): Promise<unknown> => {
+  const responseJson = await response.json();
 
   if (!response.ok) {
     throw new Error(responseJson.error || responseJson.errors);
   }
   return responseJson;
 };
 
-const fetchJSON = async (
-  input: RequestInfo,
-  init?: RequestInit
-): Promise<JSONResponse> => fetch(input, init).then(handleResponse);
+const fetchJSON = async (input: RequestInfo, init?: RequestInit) =>
```

Comment:
Why do we need `async` here? we are not using the `await` afterwards?

Reply:
Good point. Will update 👍